x11-terms/ghostty/ghostty-1.0.0.ebuild

```diff
@@ -0,0 +1,164 @@
+
+SRC_URI="
+	https://release.files.ghostty.org/${PV}/ghostty-source.tar.gz -> ${P}.tar.gz
+	${ZBS_DEPENDENCIES_SRC_URI}
+"
+S="${WORKDIR}/ghostty-source"
+
+LICENSE="
+	Apache-2.0 BSD BSD-2 BSD-4 Boost-1.0 MIT MPL-2.0
+	!system-freetype? ( || ( FTL GPL-2+ ) )
+	!system-harfbuzz? ( Old-MIT ISC icu )
+	!system-libpng? ( libpng2 )
+	!system-zlib? ( ZLIB )
+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+# TODO: simdutf integration (missing Gentoo version)
+# TODO: spirv-cross integration (missing Gentoo package)
+# TODO: glfw integration (no option from upstream)
+BDEPEND="
+	adwaita? ( gui-libs/libadwaita:1= )
+	gtk? ( gui-libs/gtk:4= )
+	man? ( virtual/pandoc )
+
+	system-fontconfig? ( >=media-libs/fontconfig-2.14.2:= )
+	system-freetype? ( >=media-libs/freetype-2.13.2:=[bzip2] )
+	system-glslang? ( >=dev-util/glslang-1.3.290.0:= )
+	system-harfbuzz? ( >=media-libs/harfbuzz-8.4.0:= )
+	system-libpng? ( >=media-libs/libpng-1.6.43:= )
+	system-libxml2? ( >=dev-libs/libxml2-2.11.5:= )
+	system-oniguruma? ( >=dev-libs/oniguruma-6.9.9:= )
+	system-zlib? ( >=sys-libs/zlib-1.3.1:= )
+"
+DEPEND="${BDEPEND}"
+
+IUSE="
+	+adwaita
+	man
+	+gtk
+	glfw
+	+pie
+
+	+system-fontconfig
+	+system-freetype
+	+system-glslang
+	+system-harfbuzz
+	+system-libpng
+	+system-libxml2
+	+system-oniguruma
+	+system-zlib
+"
+REQUIRED_USE="
+	adwaita? ( gtk )
+	^^ ( gtk glfw )
+"
+
+# XXX: Because we set --release=fast above, Zig will automatically strip
+#      the binary. Until Ghostty provides a way to disable the banner while
+#      having debug symbols we have to restrict strip to avoid
+#      "QA Notice: Pre-stripped files found" warnings.
+RESTRICT="strip"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.0.0-bzip2-dependency.patch
+	"${FILESDIR}"/${PN}-1.0.0-copy-terminfo-using-installdir.patch
+)
+
+src_configure() {
+	local my_zbs_args=(
+		-Dfont-backend=fontconfig_freetype
+		-Drenderer=opengl
+		-Dgtk-adwaita=$(usex adwaita true false)
+		-Dpie=$(usex pie true false)
+		-Demit-docs=$(usex man true false)
+		-Dversion-string="${PV}"
+
+		-f$(usex system-fontconfig sys no-sys)=fontconfig
+		-f$(usex system-freetype sys no-sys)=freetype
+		-f$(usex system-glslang sys no-sys)=glslang
+		-f$(usex system-harfbuzz sys no-sys)=harfbuzz
+		-f$(usex system-libpng sys no-sys)=libpng
+		-f$(usex system-libxml2 sys no-sys)=libxml2
+		-f$(usex system-oniguruma sys no-sys)=oniguruma
+		-f$(usex system-zlib sys no-sys)=zlib
+
+		# XXX: Ghostty displays a banner saying it is a debug build unless ReleaseFast is used.
+		--release=fast
+	)
+
+	if use gtk; then
+		my_zbs_args+=(
+			-Dapp-runtime=gtk
+		)
+	elif use glfw; then
+		my_zbs_args+=(
+			-Dapp-runtime=glfw
+		)
+	fi
+
+	zig_src_configure
+}
+
+src_install() {
+	zig_src_install
+
+	# HACK: Zig 0.13.0 build system's InstallDir step has a bug where it
+	#       fails to install symbolic links, so we manually create it
+	#       here.
+	dosym -r /usr/share/terminfo/x/xterm-ghostty /usr/share/terminfo/g/ghostty
+}
+
+pkg_postinst() {
```

Comment:
If you `s/xdg-utils/xdg` for inherit, you can drop these two phases.

Reply:
Done.